hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -885,14 +885,21 @@ public void createTenant(String tenantId, TenantArgs tenantArgs)
     final String volumeName = tenantArgs.getVolumeName();
     verifyVolumeName(volumeName);
 
+    final boolean forceCreationWhenVolumeExists =
+        tenantArgs.getForceCreationWhenVolumeExists();
+
     OmTenantArgs.Builder builder = OmTenantArgs.newBuilder();
     builder.setTenantId(tenantId);
     builder.setVolumeName(volumeName);
-    // TODO: Add more fields
-    // TODO: Include OmVolumeArgs in (Om)TenantArgs as well for volume creation?
+    builder.setForceCreationWhenVolumeExists(
+        tenantArgs.getForceCreationWhenVolumeExists());
+
+    // TODO: Add more fields. e.g. include OmVolumeArgs in (Om)TenantArgs
```

Comment:
do we need TODO ?

Reply:
Yes that is still a TODO item for volume creation customization during tenant creation. Unrelated to this item.